src/QMCWaveFunctions/WaveFunctionFactory.cpp

```diff
@@ -102,7 +102,8 @@ std::unique_ptr<TrialWaveFunction> WaveFunctionFactory::buildTWF(xmlNodePtr cur,
       if (!foundtwist)
       {
         //default twist is [0 0 0]
-        std::vector<ParticleSet::RealType> tsts(3, 0);
+        tsts.resize(3);
```

Comment:
tsts is size 3 in the declaration  (and set to 0) in the declaration.  Is there a need to set it again here?

Reply:
The original code where the declaration of 'tsts' occurs right before the use seems clear to me, but maybe I'm missing something?

Reply:
Except for the name.  In isolation, I would read 'tsts' as 'tests'.  Maybe it could be expanded to 'twists'?

Reply:
I updated the name and added a rvalue function overload to avoid the extra copy.